Tests/NIOTests/EventLoopFutureTest.swift

```diff
@@ -1207,5 +1207,63 @@ class EventLoopFutureTest : XCTestCase {
         
         try exitPromise.futureResult.wait()
     }
+ 
+    func testEventLoopFutureOrErrorNoThrow() {
+        let eventLoop = EmbeddedEventLoop()
+        let promise = eventLoop.makePromise(of: Int?.self)
+        let result: Result<Int?, Error> = .success(42)
+        promise.completeWith(result)
+
+        XCTAssertEqual(try promise.futureResult.unwrap(or: EventLoopFutureTestError.example).wait(), 42)
+    }
+ 
+    func testEventLoopFutureOrThrows() {
+        XCTAssertThrowsError(try unwrapOrErrorThatThrows())
+    }
+
+    func unwrapOrErrorThatThrows() throws {
```

Comment:
It's usually nice to make helper functions like this `private`.

Reply:
As this helper function is only in one place, can we just move it into `testEventLoopFutureOrThrows`? It's fine to make test functions throwing.